src/components/WalletSend.vue

```diff
@@ -183,6 +186,8 @@ import type { CodecString } from '@sora-substrate/sdk';
 import type { AccountAsset, AccountBalance, UnlockPeriodDays } from '@sora-substrate/sdk/build/assets/types';
 import type { Subscription } from 'rxjs';
 
+const MS_IN_DAY = 24 * 60 * 60_000;
```

Comment:
Move to consts?

Reply:
already const, don't want to make it sharable since it'll be used only here